backend/cmm_helpers.ml

```diff
@@ -1974,18 +2010,23 @@ let unaligned_set size ptr idx newval dbg =
   | Sixteen -> unaligned_set_16 ptr idx newval dbg
   | Thirty_two -> unaligned_set_32 ptr idx newval dbg
   | Sixty_four -> unaligned_set_64 ptr idx newval dbg
+  | One_twenty_eight { aligned = false } -> unaligned_set_128 ptr idx newval dbg
+  | One_twenty_eight { aligned = true } -> aligned_set_128 ptr idx newval dbg
 
 let unaligned_load size ptr idx dbg =
```

Comment:
Maybe add a comment that unaligned_set and unaligned_load are a bit of a misnomer in the 128-bit case.  We should probably let the names stand for now so we can get this merged.